packages/hub/indexing/document-context.js

```diff
@@ -23,6 +25,8 @@ module.exports = class DocumentContext {
     this._realms = [];
     this._pendingReads = [];
     this._followedRelationships = {};
+    this._routingCardId = get(upstreamDoc, 'meta.routing-card.data.id');
+    this._routingCardType = get(upstreamDoc, 'meta.routing-card.data.type');
```

Comment:
Am I understanding correctly that the meta fields are only present when the upstream document is a `spaces` resource?

If yes, I would consider moving them into being `attributes` of the `spaces` type.

Reply:
sure, sounds good.